src/Containers/DenseAxisArray.jl

```diff
@@ -348,6 +348,19 @@ function Base.IndexStyle(::Type{DenseAxisArray{T,N,Ax}}) where {T,N,Ax}
     return IndexAnyCartesian()
 end
 
+function Base.setindex!(
+    A::DenseAxisArray{T,N},
+    value::DenseAxisArray{T,N},
+    args...,
+) where {T,N}
+    @show args
+    @show Base.to_index(A, args)
```

Comment:
Remove

Reply:
Oops. Yeah. I guess my bigger question was;

> I'm not overly convinced of this approach.
> should you be able to access a value that isn't in the view, but is in the underlying array?

Reply:
I don't think we should give access to it